.github/workflows/CI.yml

```
@@ -0,0 +1,54 @@
+
+    steps:
+      # 1. Checkout the repository
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      # 2. Setup .NET SDK
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: '8.0.x'
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)